orange-datamodel/src/main/java/com/hartwig/hmftools/datamodel/isofox/RnaFusion.java

```diff
@@ -8,11 +8,17 @@
 @Gson.TypeAdapters
 @Value.Immutable
 @Value.Style(passAnnotations = { NotNull.class, Nullable.class })
-public interface RnaFusion {
-
+public interface RnaFusion
+{
     @NotNull
     String name();
 
+    @NotNull
+    String geneStart();
+
+    @NotNull
+    String geneEnd();
+
```

Comment:
It's not ideal but I would rephrase this to geneUp / geneDown, since all other fields in RnaFusion are called up/down already. Alternatively we could rename all fields to the "start/end" convention to make RnaFusion consistent with LinxFusion